datacube_stats/statistics.py

```diff
@@ -358,6 +358,63 @@ def measurements(self, input_measurements):
         ]
 
 
+class MedNdwi(Statistic):
+    import warnings
+
+    """Calculate ndwi and then median image through time"""
+
+    def compute(self, data):
+        # This is a special case to implement, after calculating ndwi as med.
+        # Finally median through time for ITEM product
+        med = (data.green - data.nir) / (data.green + data.nir)
+        # stop all bad data and reset to the following
+        with warnings.catch_warnings():  # Don't print error for not knowing nan
+            warnings.simplefilter("ignore")
+            med.values[med.values < -1] = np.nan
+            med.values[med.values > 1] = np.nan
+            med.values[med.values == 10] = np.nan
+            med = med.median(dim='time', keep_attrs=True, skipna=True)
+        return med.to_dataset(name='ndwi')
+
+    def measurements(self, input_measurements):
+        return [
+            {
+                'name': 'ndwi',
+                'dtype': 'float32',
+                'nodata': np.nan,
+                'units': '1'
+            }
+        ]
+
+
+class StdNdwi(Statistic):
```

Comment:
See the comment on the configuration file. I don't think this class needs to exist.

Also, this class is almost exactly identical to `MedNdwi`, it's pretty poor form to duplicate code so egregiously. Unless it really makes things simpler, which I don't think is the case here.